optex/base/output.opm

```diff
@@ -214,11 +204,11 @@
    \_fontdef\_draftfont{\_setfontsize{at10pt}\_bf}%
    \_global\_let\_draftfont=\_draftfont
 }
-\_def \_draftbox #1{\_setbox0=\_hbox{#1}%
+\_def \_draftbox #1{\_setbox0=\_hbox{\_setgreycolor{.8}#1}%
    \_kern.5\_vsize \_kern\_voffset \_kern4.5\_wd0
    \_hbox to0pt{\_kern.5\_xhsize \_kern\_hoffset \_kern-2\_wd0
    \_pdfsave \_pdfrotate{55}\_pdfscale{10}{10}%
-   \_hbox to0pt{\_localcolor\_setgreycolor{.8}\_box0\_hss}%
+   \_hbox to0pt{\_box0\_hss}%
    \_pdfrestore
    \_hss}%
```

Comment:
Can this be improved?

Reply:
I plan to re-implement `\draftbox` to keep zero vertical dimensions and do `\gtoksapp\_pgbackground` instead `\_pgbackground=`. Moreover, I add a recommendation to the doc.: use `\gtoksapp` or `\gtokspre` if you have your own background and keep vertical size zero. Then you can have more layers of the page background.